arrows/ocv/image_container.h

```diff
@@ -54,9 +54,10 @@ class KWIVER_ALGO_OCV_EXPORT image_container
 {
 public:
 
+  enum ColorMode{RGB,BGR};
+
   /// Constructor - from a cv::Mat
-  explicit image_container(const cv::Mat& d)
-  : data_(d) {}
+  explicit image_container(const cv::Mat& d, ColorMode cm = RGB);
```

Comment:
I agree with @mattdawkins that since there is a default color mode should probably be BGR here, because we assume by default that cv::Mat images are in BGR.